src/PlaywrightSharp/ContextEvent.cs

```diff
@@ -4,19 +4,22 @@
 namespace PlaywrightSharp
 {
     /// <summary>
-    /// Context events. See <see cref="IBrowserContext.WaitForEventAsync{T}(PlaywrightEvent{T}, Func{T, bool}, int?)"/>.
+    /// Context events. See <see cref="IBrowserContext.WaitForEventAsync(string, float?)"/>.
```

Comment:
I think we could still point to the other function.

Reply:
Yes, now we can, but I've removed it before in the attempt to get rid of the awesomeness that is `PlaywrightEvent<>` :)